src/library/library.h

```diff
@@ -115,6 +128,10 @@ class Library: public QObject {
     // emit this signal to enable/disable the cover art widget
     void enableCoverArtDisplay(bool);
     void trackSelected(TrackPointer pTrack);
+#ifdef __ENGINEPRIME__
+    void exportLibrary();
```

Comment:
The naming is very generic although this "export" feature only supports Engine Prime.

Reply:
No problem to rename - the djinterop library may one day support other formats (meaning this could be a more general-purpose export), but it's not there yet.  Will change.

Reply:
Even then we probably want options to export the different formats separately.

Reply:
I've now pushed a 'cleanup' commit that changes the visible UI labels to specifically mention Engine Prime, as that's what the user will most obviously understand.

However, I'd like to check opinions on whether it's also worth refactoring/renaming methods/classes that are not Engine Prime-specific.  My "longer-term" ideas for this when coming up with the class design were:

* The `DlgLibraryExport` class today has a UI layout that is specific to Engine Prime, and is labelled as being an Engine Prime export.  But in future, it might allow the user to choose their export format (e.g. Engine Prime, Pioneer, etc).
  * It would allow a consistent view of things common to any export (e.g. choice of crates).
  * It would also allow the choice of different format-specific export options as @Holzhaus mentions above.
* The `DlgLibraryExport` class today has only a `startEnginePrimeExport` signal, but in future it might also have a `startPioneerExport` signal, etc. depending on what type of export the user chooses.
* The `LibraryExporter` class launches the dialog, and connects `startEnginePrimeExport` to an invocation of the `EnginePrimeExportJob` worker.  In future, it might connect `startPioneerExport` to a theoretical `PioneerExportJob` worker too, and so on.

So the idea is that `DlgLibraryExport` and `LibraryExporter` are not format-specific, but serve to launch the eventual format-specific export job that the user chooses.

If I've been able to convince you that this arrangement might be of benefit in the future, the PR is (hopefully!) in good shape already.  If we prefer to tackle the multi-format problem later, I would probably want to rebase the PR, as the changes would be a bit larger.